metering/reports/metering-storage-locations.adoc

```diff
@@ -0,0 +1,76 @@
+[id="metering-storage-locations"]
+= Storage Locations
+include::modules/common-attributes.adoc[]
+:context: metering
+
+toc::[]
+
+A StorageLocation is a custom resource that configures where data will be stored by the reporting-operator. This includes the data collected from Prometheus, and the results produced by generating a Report custom resource.
+
+You only need to configure a StorageLocation if you want to store data in multiple locations, like multiple S3 buckets or both S3 and HDFS, or if they wish to access a database in Hive/Presto that was not created by metering. For most users this is not a requirement, and the xref:../../metering/configuring-metering/metering-about-configuring.adoc#metering-about-configuring[documentation on configuring metering] is sufficent to configure all necessary storage components. 
+
+== Example StorageLocation
+
+This first example is what the built-in local storage option looks like. It is configured to use Hive, and by default data is stored wherever Hive is configured to use storage (HDFS, S3, or a ReadWriteMany PVC).
```

Comment:
I think these examples to should be reformatted for easier viewing. Take a look at our storage examples here: https://docs.openshift.com/container-platform/4.1/storage/understanding-persistent-storage.html#block-volume-examples_understanding-persistent-storage